kwc-numpad.js

```diff
@@ -159,5 +160,18 @@ class KwcNumpad extends PolymerElement {
     _valueChanged() {
         this.stringValue = typeof this.value === 'undefined' ? '0' : this.value.toString();
     }
+    _setWidth() {
```

Comment:
This is very hacky. Can't you force the width from CSS?

Reply:
Yeah I know, but you won't touch again.
Give me a while to check again with CSS